### DISCUSSION TITLE
New PayPal connection instructions

### DISCUSSION THREAD

**Discussion by DieselEffect** [gameserverapp/Platform#868] [Suggestions]
With this process fresh in my mind

1. Login to the PayPal account you wish to have funds deposited to.
2. Navigate to https://developer.paypal.com/dashboard/applications/live
3. Ensure the toggle at the top-right is set to 'Live' and not 'Sandbox'.
4. Near the top, around REST API Apps, click 'Create App'.
5. Name your app and click 'Create App'.
6. Copy 'Client ID' and 'Secret key 1' and enter them here and click 'Save'
7. Scroll down to 'Features', uncheck all unnecessary boxes and click 'Save Changes'.
8. Scroll down to 'Live Webhooks' and click 'Add Webhook'.
9. Enter the Webhook URL shown on the right side of this page.
10. Expand the Payments & Payouts category and check the box for "Payment sale completed".
11. Scroll to the bottom and click 'Save'

I'm not sure if step 6 needs to be placed at the bottom or not, but these are the steps needed with PayPal's updated interface.

**Comment by DieselEffect** [comment]
This still needs added to the in-dashboard instructions at https://dash.gameserverapp.com/monetization/settings/paypal

**Comment by mvaessen** [comment]
This was updated, thanks for the info!
